scss/_patterns_chip.scss

```diff
@@ -73,6 +73,19 @@
 
 // Default notification styling
 @mixin vf-chips-default {
+  // stylelint-disable selector-max-type -- allow button selector for interactive chips
+
+  // Status-color chips all use the button pattern mixin to set their colors, so they already have most button styling.
+  // We need to add the default button styling to the link default chip as well.
+  a.p-chip {
+    @include vf-button-pattern;
```

Comment:
Would it make sense to make `.p-chip` include button styles by default, regardless of the type of element?

Reply:
I think we discussed but never implemented chips that are not clickable, and do not have an affordance that satisfies 3:1 contrast (meaning the border). So from that perspective, I'd say we shoudln't.

Reply:
possible caveat - if wcag has anything on chips, and clickability is mandatory in their spec, that would change things.

Reply:
@bartaz Good catch, I've updated this to unify link chip and button chip styling